samples/javascript_nodejs/46.teams/teamsActivityHandler.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityHandler, ActivityTypes } = require('botbuilder');
+
+class TeamsActivityHandler extends ActivityHandler {
+
+    constructor() {
+        super();
+        this.onUnrecognizedActivityType(async (context, next) => {
```

Comment:
I know we've avoided comments, but I feel like this a good time to explain what we're doing here.